### PR TITLE
Add database migrations

### PR DESCRIPTION
**Pull request by wistefan**
Change primary keys to combined keys and add migrations via liquibase.

### CODE REVIEW

File: doc/manuals-ld/troe.md [hunk 1]
```diff
@@ -7,3 +7,50 @@ The sink used for TRoE is Postgres, with PostGIS and TimescaleDB extensions.
...
+[timescaledb-postgis image](https://hub.docker.com/layers/timescale/timescaledb-postgis/latest-pg12/images/sha256-40be823de6035faa44d3e811f04f3f064868ee779ebb49b287e1c809ec786994?context=explore).
+
+You can start it with 
+```
+docker run -e POSTGRES_USER=orion -e POSTGRES_PASSWORD=orion -e POSTGRES_HOST_AUTH_METHOD=trust timescale/timescaledb-postgis:latest-pg12
+```
+
+## Database migration
+
+You can find the databse migration scripts ins the [database-folder](../../database)
```
Comment (kzangeli) [inline doc/manuals-ld/troe.md]: Typo: databse.

Also, we don't want any 'we' and 'you' in the documentation, except for "special occasions".
 I'll go over it shortly.

File: test/functionalTest/harnessFunctions.sh [hunk 2]
```diff
@@ -1294,7 +1294,7 @@ function orionCurl()
   then
     echo "Broker seems to have died ..."
   else
-    cat /tmp/httpHeaders.out  | sed 's///g' > /tmp/httpHeaders.noCtrlM
+    cat /tmp/httpHeaders.out  | sed 's/^M//g' > /tmp/httpHeaders.noCtrlM
```
Comment (kzangeli) [inline test/functionalTest/harnessFunctions.sh]: Seems like your editor is unable to deal with ASCII control characters.
I'll fix it in emacs as part of the PR.

^M is supposed to be one char, not two.

File: doc/manuals-ld/troe.md [hunk 3]
```diff
@@ -7,3 +7,50 @@ The sink used for TRoE is Postgres, with PostGIS and TimescaleDB extensions.
 While Orion-LD takes care of populating the TRoE databases, another component handles the queries of temporal data - [Mintaka](https://github.com/FIWARE/Mintaka).
 
 So, for queries of the temporal data, instead of sending the requests to Orion-LD, on (default) port 1026, the queries are sent to Mintaka, on (default) port xxxx.
```
Comment (kzangeli) [inline doc/manuals-ld/troe.md]: What's the default port of Mintaka?

**Review by kzangeli — CHANGES_REQUESTED** [review]
Great job!
Just, let me adjust the English a little in troe.md.
And, perhaps we should use "initial.sql" in harnessFunctions.sh and in the C++ source code as well - making the makefile create a C++ file based on it. How's your awk? :)

File: doc/manuals-ld/troe.md (hunk 3 above)
Comment (wistefan) [reply doc/manuals-ld/troe.md]: 8080, I'll update.

File: test/functionalTest/harnessFunctions.sh (hunk 2 above)
Comment (wistefan) [reply test/functionalTest/harnessFunctions.sh]: This would be great, I was trying a lot to get it back correctly:)

**Comment by wistefan** [comment]
> Great job!
> Just, let me adjust the English a little in troe.md.
> And, perhaps we should use "initial.sql" in harnessFunctions.sh and in the C++ source code as well - making the makefile create a C++ file based on it. 

Yes, that would be good. 

>How's your awk? :)
Pretty much non-existent.

File: test/functionalTest/harnessFunctions.sh (hunk 2 above)
Comment (kzangeli) [reply test/functionalTest/harnessFunctions.sh]: :)

File: doc/manuals-ld/troe.md (hunk 1 above)
Comment (kzangeli) [reply doc/manuals-ld/troe.md]: Fixed in 0b6b6e2

File: test/functionalTest/harnessFunctions.sh (hunk 2 above)
Comment (kzangeli) [reply test/functionalTest/harnessFunctions.sh]: Fixed in 0b6b6e2

File: doc/manuals-ld/troe.md (hunk 3 above)
Comment (kzangeli) [reply doc/manuals-ld/troe.md]: Fixed in 0b6b6e2

**Review by kzangeli — APPROVED** [review]
LGTM

**Pull request merged**